guides/v2.0/frontend-dev-guide/css-topics/css-preprocess.md

```diff
@@ -161,6 +161,8 @@ If you need to import a remote CSS file in your `.less` source, use `url()` nota
 @import url('//fonts.googleapis.com/css?family=Titillium+Web:400,300,200,600.css');
 {%endhighlight%}
 
+TO include the <a href="{{ page.baseurl }}/frontend-dev-guide/css-topics/using-fonts.html">font in your theme's</a> CSS files using the `@font-face` CSS rule for the fastest loading time.
```

Comment:
Here are some suggested edits. If you agree, feel free to merge this change. Thanks!

```suggestion
To [include the font]({{ page.baseurl }}/frontend-dev-guide/css-topics/using-fonts.html) in your theme's CSS files, use the `@font-face` CSS rule for the fastest loading time.
```